src/app/views/query-response/headers/index.ts

```diff
@@ -1,2 +1,2 @@
 import ResponseHeaders from './ResponseHeaders';
-export { ResponseHeaders };
+export default ResponseHeaders;
```

Comment:
could `export { ResponseHeaders as default };`  also work?

Reply:
Yes. Let me change this